cmd/jujud/agent/machine_test.go

```diff
@@ -1052,7 +1054,7 @@ func (s *MachineSuite) TestCertificateUpdateWorkerUpdatesCertificate(c *gc.C) {
 				close(updated)
 				break
 			}
-			time.Sleep(10 * time.Millisecond)
+			time.Sleep(100 * time.Millisecond)
```

Comment:
Becase the test takes ages anyway and checking every 10ms is checking way too often. 100ms is fine.